src/libclipper/include/clipper/metrics.hpp

```diff
@@ -291,6 +291,7 @@ class Histogram : public Metric {
   void insert(const int64_t value);
   const HistogramStats compute_stats();
   static double percentile(std::vector<int64_t> snapshot, double rank);
+  double percentile(double rank);
```

Comment:
Looks good! Can you add documentation indicating that a snapshot is obtained from the histogram's sampler and then used to calculate the percentile?